Index: src/main/java/org/uz/entiti/entitiy/User.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.uz.entiti.entitiy;\r\n\r\nimport lombok.AllArgsConstructor;\r\nimport lombok.Data;\r\nimport lombok.NoArgsConstructor;\r\n\r\n@Data\r\n@AllArgsConstructor\r\n@NoArgsConstructor\r\npublic class User {\r\n    private String id;\r\n    private String name;\r\n    private String surname;\r\n    private String phoneNumber;\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/uz/entiti/entitiy/User.java b/src/main/java/org/uz/entiti/entitiy/User.java
--- a/src/main/java/org/uz/entiti/entitiy/User.java	
+++ b/src/main/java/org/uz/entiti/entitiy/User.java	
@@ -8,8 +8,9 @@
 @AllArgsConstructor
 @NoArgsConstructor
 public class User {
-    private String id;
     private String name;
     private String surname;
     private String phoneNumber;
+    private String password;
+    private Role role;
 }
Index: src/main/java/org/uz/entiti/service/AuthService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.uz.entiti.service;\r\n\r\npublic class AuthService {\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/uz/entiti/service/AuthService.java b/src/main/java/org/uz/entiti/service/AuthService.java
--- a/src/main/java/org/uz/entiti/service/AuthService.java	
+++ b/src/main/java/org/uz/entiti/service/AuthService.java	
@@ -1,5 +1,39 @@
 package org.uz.entiti.service;
 
+import org.uz.entiti.entitiy.Role;
+import org.uz.entiti.entitiy.User;
+
+import static org.uz.entiti.Database.DB.strScanner;
+import static org.uz.entiti.Database.DB.users;
+
 public class AuthService {
+    public static void signUp() {
+        User user = new User();
+
+        System.out.println("Enter name...");
+        user.setName(strScanner.nextLine());
+        System.out.println("Enter surname...");
+        user.setSurname(strScanner.nextLine());
+        System.out.println("Enter phone number...");
+        user.setPhoneNumber(strScanner.nextLine());
+        System.out.println("Enter password...");
+        user.setPassword(strScanner.nextLine());
+        user.setRole(Role.USER);
+        users.add(user);
+        System.out.println("Signed successfully");
+    }
+
+    public static void signIn() {
+        System.out.println("Enter name...");
+        String name = strScanner.nextLine();
+        System.out.println("Enter password...");
+        String password = strScanner.nextLine();
+        for (int i = 0; i < users.size(); i++) {
+            if (users.get(i).getName().equals(name) && users.get(i).getPassword().equals(password)) {
+                if (users.get(i).getRole().equals(Role.USER)) UserService.userService();
+                if (users.get(i).getRole().equals(Role.ADMIN)) AdminService.adminService();
+            }
+        }
+    }
 
 }
Index: src/main/java/org/uz/entiti/service/AdminService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.uz.entiti.service;\r\n\r\npublic class AdminService {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/uz/entiti/service/AdminService.java b/src/main/java/org/uz/entiti/service/AdminService.java
--- a/src/main/java/org/uz/entiti/service/AdminService.java	
+++ b/src/main/java/org/uz/entiti/service/AdminService.java	
@@ -1,4 +1,81 @@
 package org.uz.entiti.service;
 
+import org.uz.entiti.Database.DB;
+
 public class AdminService {
+    public static void adminService() {
+        while (true) {
+            System.out.println("""
+                    Admin Menu:
+                    1. Show Rooms
+                    2. Add Room
+                    3. Remove Room
+                    4. Show Bookings
+                    5. Remove Booking
+                    6. Show Users
+                    7. Remove User
+                    8. Exit to Main Menu
+                    """);
+
+            int choice = DB.scanner.nextInt();
+            DB.scanner.nextLine(); // Consume newline character
+
+            switch (choice) {
+                case 1:
+                    showRooms();
+                    break;
+                case 2:
+                    addRoom();
+                    break;
+                case 3:
+                    removeRoom();
+                    break;
+                case 4:
+                    showBookings();
+                    break;
+                case 5:
+                    removeBooking();
+                    break;
+                case 6:
+                    showUsers();
+                    break;
+                case 7:
+                    removeUser();
+                    break;
+                case 8:
+                    return;
+                default:
+                    System.out.println("Invalid option. Please try again.");
+            }
+
+        }
+    }
+
+    private static void showRooms() {
+
+    }
+
+    private static void addRoom() {
+
+    }
+
+    private static void removeRoom() {
+
+    }
+
+    private static void showBookings() {
+
+    }
+
+    private static void removeBooking() {
+
+    }
+
+    private static void showUsers() {
+
+    }
+
+    private static void removeUser() {
+
+    }
 }
Index: src/main/java/org/uz/entiti/service/UserService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.uz.entiti.service;\r\n\r\npublic class UserService {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/uz/entiti/service/UserService.java b/src/main/java/org/uz/entiti/service/UserService.java
--- a/src/main/java/org/uz/entiti/service/UserService.java	
+++ b/src/main/java/org/uz/entiti/service/UserService.java	
@@ -1,4 +1,60 @@
 package org.uz.entiti.service;
 
+import org.uz.entiti.Database.DB;
+
 public class UserService {
+    public static void userService() {
+        while (true) {
+
+            System.out.println("""
+                    0 -> exit
+                    1 -> my booking
+                    2 -> active rooms
+                    3 -> booking
+                    4 -> cancel
+                    5 -> balance
+                    """);
+            switch (DB.scanner.nextInt()) {
+                case 0 -> {
+                    System.out.println("bye bye");
+                    return;
+                }
+                case 1 -> {
+                    myBookings();
+                }
+                case 2 -> {
+                    activeRooms();
+                }
+                case 3 -> {
+                    booking();
+                }
+                case 4 -> {
+                    cancel();
+                }
+                case 5 -> {
+                    balance();
+                }
+            }
+        }
+    }
+
+    private static void myBookings() {
+
+    }
+
+    private static void activeRooms() {
+
+    }
+
+    private static void booking() {
+
+    }
+
+    private static void cancel() {
+
+    }
+
+    private static void balance() {
+
+    }
 }
